src/rust/src/rustzcash.rs

```diff
@@ -1065,6 +1069,23 @@ pub extern "C" fn librustzcash_zip32_xsk_derive(
         .expect("should be able to serialize an ExtendedSpendingKey");
 }
 
+/// Derive the Sapling internal spending key from the external extended
+/// spending key
+#[no_mangle]
+pub extern "C" fn librustzcash_zip32_xsk_derive_internal(
```

Comment:
This should have `sapling` in the name.
```suggestion
pub extern "C" fn librustzcash_zip32_sapling_xsk_derive_internal(
```